package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-big-calendar",
+  "name": "ehahn9-react-big-calendar",
```

Comment:
fix

Reply:
yeah, my inexperience showing... had to npm publish this branch but will revert the name in the next attempt.

Reply:
you shouldn't need to do anything with npm here :) whatever is on npm is unrelated to what's in the PR here. you can commit a change to your branch and push and it'll update without needing to publish anything